link.go

```diff
@@ -53,21 +53,47 @@ unlinked QmVGtdTZdTFaLsaj2RwdVG8jcjNNcp1DE914DKZ2kHmXHw /home/user/go/src/github
 			Name:  "a,all",
 			Usage: "Remove all existing symlinks and reinstate the gx packages. Use with -r.",
 		},
+		cli.StringFlag{
```

Comment:
I wouldn't add this flag. Instead, we could add a top-level `-C` flag (like `git`) but we can probably do that later.